boards/anavi/anavi-arrows/code.py

```diff
@@ -0,0 +1,92 @@
+import board
+
+from arrows import AnaviArrows
+
+from kmk.extensions.LED import LED
+from kmk.extensions.media_keys import MediaKeys
+from kmk.extensions.peg_oled_Display import (
+    Oled,
+    OledData,
+    OledDisplayMode,
+    OledReactionType,
+)
+from kmk.extensions.RGB import RGB, AnimationModes
+from kmk.keys import KC
+from kmk.modules.encoder import EncoderHandler
+
+print('ANAVI Arrows')
```

Comment:
Unecessary print statement. If you think this is usefull for debugging, use the debug utility.
```suggestion
```